src/SFA.DAS.AssessorService.Web.Staff/Views/CertificateApprovals/Approved.cshtml

```diff
@@ -0,0 +1,129 @@
+﻿@model SFA.DAS.AssessorService.Web.Staff.ViewModels.Private.CertificateApprovalViewModel
+@using SFA.DAS.AssessorService.Web.Staff
+
+
+@{ ViewBag.Title = "Staff Dashboard"; }
+
+<div>
+    <nav class="app-navigation govuk-clearfix">
+        <ul class="app-navigation__list">
+            <li class="app-navigation">
+                <a asp-controller="CertificateApprovals" asp-action="New">New</a>
+            </li>
+            <li class="app-navigation">
+                <a asp-controller="CertificateApprovals" asp-action="SentForApproval">Sent For Approval</a>
+            </li>
+            <li class="app-navigation--current-page">
+                <a asp-controller="CertificateApprovals" asp-action="Approved">Approved</a>
+            </li>
+            <li class="app-navigation">
+                <a asp-controller="CertificateApprovals" asp-action="Rejected">Rejected</a>
+            </li>
+        </ul>
+    </nav>
+</div>
+
+<div class="grid-row">
+    <div class="column-full">
+        <form class="apprentice-approvals-form js-disable-on-submit" type="post" asp-controller="CertificateApprovals" asp-action="Approvals">
+            <a class="link-back" asp-controller="Dashboard" asp-action="Index">Return to dashboard</a>
+
+            @* hidden with css by default *@
+            <div class="js-error-summary error-summary" role="alert" aria-labelledby="error-summary-heading" tabindex="-1">
+                <h2 class="heading-medium error-summary-heading" id="error-summary-heading">
+                    There is a problem
+                </h2>
+                <ul class="js-error-summary-list error-summary-list"></ul>
+            </div>
+
+            @if (!ViewData.ModelState.IsValid)
+            {
+                <div class="error-summary" role="alert" aria-labelledby="error-summary-heading" tabindex="-1">
+                    <h2 class="heading-medium error-summary-heading" id="error-summary-heading">
+                        There is a problem
+                    </h2>
+                    <ul class="error-summary-list"></ul>
+                </div>
+            }
+
+                <div sfa-anyvalidationerror-class="after-error-summary">
+
+                    @{
+                        var count = Model.ApprovedCertificates.Items.Count();
+                        var approvalResults =
+                            new SFA.DAS.AssessorService.Web.Staff.ViewModels.Private.CertificateDetailApprovalViewModel[count];
+                        for (int i = 0; i < count; i++)
+                        {
+                            approvalResults[i] = new SFA.DAS.AssessorService.Web.Staff.ViewModels.Private.CertificateDetailApprovalViewModel();
+                        }
```

Comment:
This code needs to be in the VM preferably.

Reply:
Refactored and moved to VM also done it for other cshtml